images/ubuntu2004/scripts/cloudinit_install.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+GRUB_CMDLINE_LINUX_DEFAULT="console=tty1 console=ttyS0,115200n8 earlyprintk=ttyS0,115200 rootdelay=300 quiet splash"
```

Comment:
This sets a shell variable and isn't going to really do much. I think this needs to be written to `/etc/default/grub` and then run `update-grub`. Data probably already exists in the file, so this will probably need to be a `sed` command to update/replace instead of append

https://docs.microsoft.com/en-us/azure/virtual-machines/linux/create-upload-ubuntu

Reply:
Thanks, changing this via sed and calling update-grub now